python/hail/methods/statgen.py

```diff
@@ -2933,10 +2933,9 @@ def ld_prune(call_expr, r2=0.2, bp_window_size=1000000, memory_per_core=256, kee
     --------
     Filter a dataset to those variants returned by ld_prune.
```

Comment:
Suggestion: "Filter a dataset to those variants given by LD pruning its biallelic variants."

And right after the code: "Filtering to biallelic variants is unnecessary if the dataset is already biallelic." (unless you think that's too obvious)